### DISCUSSION TITLE
[Dropdown] "refresh" and "refreshItems" are not working

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#2033] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hello, I need to change an option status after an async call but I can't get it updated in the dropdown.

```
$('select').dropdown();

setTimeout(() => {
   $('select option[value="3"]').attr('disabled', true);
   $('select').dropdown('refreshItems');
}, 3000);
```

I have to use `destroy` and then reinstantiate it to make it working:

```
$('select option[value="3"]').attr('disabled', true);
$('select').dropdown('destroy');
$('select').dropdown();
```

Here is a JSFiddle: https://jsfiddle.net/6y8d4qxo/2/

Is anything wrong with my usage?

**Comment by lubber-de** [comment, marked as answer]
This is a bug, because the select observer should detect changes to the select tag and recreate the menu automatically.
Unfortunately it did not detect attribute changes to option tags.

I fixed this by #2034 
See https://jsfiddle.net/lubber/hjdv6bnL/9/

**Reply by smartm0use** [reply]
I see, so I was wrong posting it here as discussion instead of issue.
Thank you for your prompt fix!
